src/modules/Accordion/AccordionTitle.js

```diff
@@ -43,17 +50,29 @@ export default class AccordionTitle extends Component {
     parent: 'Accordion',
   }
 
+  static create = createShorthandFactory(AccordionTitle, content => ({ content }))
+
   handleClick = (e) => {
     const { onClick } = this.props
 
     if (onClick) onClick(e, this.props)
   }
 
+  renderContent(content) {
+    return isValidElement ? content : (
```

Comment:
Oh my. I've fixed isValidElement issue and left the logic intact (apart from inserting the `div`) as I strongly believe users should have the ability to customise the title fully. Without the check for element if I pass
```
title: (
   <div>
      <Icon name='heart' />
      What a lovely title
   </div>
)
```
it will render two icons (dropdown and heart). On the other side when somebody passes text the dropdown icon should be rendered.